.github/workflows/smoketests.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
             - name: Cypress run
               # Uses the official Cypress GitHub action https://github.com/cypress-io/github-action
-              if: "!contains(steps.pr_information.outputs.branch_name, 'translation-new-homepage')"
+              if: "!contains(steps.pr_information.outputs.branch_name, 'translation')"
```

Comment:
Hi @markodonnell-deriv , it can't be translation, As all the clickup cards that needs translation will start with translation keyword

Reply:
ok, could we rename the branches to 'skipsm' or do we go back to the idea of using labels?